.github/workflows/scripts/get_dependencies.sh

```diff
@@ -236,6 +246,7 @@ for depend in "$@"; do
     get_clang_format
   elif [ "${depend}" = "cmake" ]; then
     get_cmake "${cmake_version}"
+    get_ninja
```

Comment:
I don't like the idea of coupling dependencies in this script (especially before we know that all of the repos build with ninja). I think it's better to have each repo list ninja as a dependency and install it before running CMake.

Reply:
Ok, this would require to update build script so that it doesn't fail if `ninja` was not used. Addressed with 5dc37982442231181868968cebef8b48a2954f7d.